internal/querynode/meta_replica.go

```diff
@@ -477,6 +481,31 @@ func (replica *metaReplica) getSegmentIDs(partitionID UniqueID, segType segmentT
 	return replica.getSegmentIDsPrivate(partitionID, segType)
 }
 
+// getGrowingSegmentNumByVChannel returns number of segments which virtual channel is vChannel
+// if partitionIDs is empty, it means that filtering by partitionIDs is not required.
+func (replica *metaReplica) getGrowingSegmentNumByVChannel(vChannel Channel, partitionIDs []UniqueID) int {
```

Comment:
I thought we should extend getSegmentIDsByVChannel to support get Streaming segmentIds?
There is no reason to add a new interface for get count